tox.ini

```diff
@@ -24,11 +24,12 @@ deps =
     hypothesis==6.21.6
     py-eth-sig-utils==0.4.0
     aiohttp==3.7.4.post0
-    open-autonomy==0.2.1
-    open-aea[all]==1.17.0
-    open-aea-ledger-ethereum==1.17.0
-    open-aea-ledger-cosmos==1.17.0
-    open-aea-cli-ipfs==1.17.0
+    open-autonomy==0.2.2
```

Comment:
wrong versions

Reply:
Ok, I copied the whole file from AA4 since there were a lot of changes. Updating to latest.

Reply:
https://github.com/valory-xyz/agent-academy-4/blob/main/tox.ini then you are behind main